src/app/lib/utilities/getUrlHelpers/getEmbedUrl/index.test.js

```diff
@@ -283,3 +283,87 @@ describe('makeAbsolute makes a relative URL absolute', () => {
     delete process.env.SIMORGH_APP_ENV;
   });
 });
+
+describe('buildAvEmbedURL', () => {
+  describe('service with variant', () => {
+    it('should build an embed URL for the page', () => {
+      expect(
+        getEmbedUrl({
+          mediaId: '/serbian/cyr/av-embeds/srbija-68707945',
+          type: 'avEmbed',
+          queryString: '',
+        }),
+      ).toEqual('https://www.bbc.com/serbian/cyr/av-embeds/srbija-68707945');
+    });
+
+    it('should build an embed URL for the page omitting renderer env', () => {
+      expect(
+        getEmbedUrl({
+          mediaId: '/russian/av-embeds/features-49881797?renderer_env=live',
+          type: 'avEmbed',
+          queryString: '',
+        }),
+      ).toEqual('https://www.bbc.com/russian/av-embeds/features-49881797');
+    });
+
+    it('should build an embed with a vpid URL for the page', () => {
+      expect(
+        getEmbedUrl({
+          mediaId: '/serbian/cyr/av-embeds/srbija-68707945/vpid/p0cfmdwj',
+          type: 'avEmbed',
+          queryString: '',
+        }),
+      ).toEqual(
+        'https://www.bbc.com/serbian/cyr/av-embeds/srbija-68707945/vpid/p0cfmdwj',
+      );
+    });
+
+    it('should build an embed with a pid URL for the page', () => {
+      expect(
+        getEmbedUrl({
+          mediaId: '/serbian/cyr/av-embeds/srbija-68707945/pid/p0cfmdwn',
+          type: 'avEmbed',
+          queryString: '',
+        }),
+      ).toEqual(
+        'https://www.bbc.com/serbian/cyr/av-embeds/srbija-68707945/pid/p0cfmdwn',
+      );
+    });
+  });
+
+  describe('service without variant', () => {
+    it('should build an embed URL for the page', () => {
+      expect(
+        getEmbedUrl({
+          mediaId: '/mundo/av-embeds/srbija-68707945',
```

Comment:
Do we a real mundo (this URL has srbija in it)?

Also, what's the deal with embeds from Optimo pages - are the video IDs in this format, or are they different? If they are hugely different to the CPS format, is worth adding another test just to make sure the URL is as expected?

Reply:
Cool I have updated to match real non-variant assets (on russian). I've updated the wording of the tests to be more general. (I think before they specific to the function being used to contstruct the metadata og:url)

And I'm not too sure RE Optimo/ CPS differences sorry. This work supports Syndication/ CPS assets. In general terms, we think that the aresMedia structure between CPS and Optimo should mostly be the same though, so if we build support for CPS, Optimo should kinda just work if we introduce routing for it.